pkg/core/mempool/mem_pool.go

```diff
@@ -26,6 +27,7 @@ var (
 // item represents a transaction in the the Memory pool.
 type item struct {
 	txn        *transaction.Transaction
+	blockStamp uint32
 	timeStamp  time.Time
```

Comment:
Just drop it. Or maybe even just backport 5df726db68afc87bfbac11194b3f925ce4efd2fd.

Reply:
Fixed.